### ISSUE TITLE
Translate to Spanish and create tutorials

### ISSUE THREAD

**Issue by RafaMartorell** [olive-editor/olive#1000]
I offer to help in the translation into Spanish and to write some tutorials and also make video tutorials about this application because I am really enjoying it.

I do not understand programming but if I can help in translating and teaching the use of the application, I am a professional in the audiovisual world and an admirer of open source projects.

I consider it a very good job, I am anxious to know the stable version :-)

**Comment by oluja1973** [comment]
Hi Rafa,

here you can see how to help translating in your language. https://github.com/olive-editor/olive/wiki/Translations

Regards, Oluja!

**Comment by RafaMartorell** [comment]
Thank you very much, with these instructions it has been very easy for me to be able to translate.

Sorry for the delay but some work issues prevented me from facing this task until yesterday.

I am working on it and soon it will be completely finished.

A pleasure for me to help and contribute my bit to this fabulous project.
________________________________
De: oluja1973 <notifications@github.com>
Enviado: sábado, 15 de junio de 2019 21:37
Para: olive-editor/olive
Cc: RafaMartorell; Author
Asunto: Re: [olive-editor/olive] Translate to Spanish and create tutorials (#1000)


Hi Rafa,

here you can see how to help translating in your language. https://github.com/olive-editor/olive/wiki/Translations

Regards, Oluja!

**Comment by RafaMartorell** [comment]
I finished the translation, I am not very expert in git but I think I have taken the correct steps to update the translation into Spanish.

Attachment ts file in the message or else this can be found here: https://drive.google.com/file/d/1fMWHvA2iA6Yr-fKfRknYBMNyDTGEwOkU/view?usp=sharing
________________________________
De: oluja1973 <notifications@github.com>
Enviado: sábado, 15 de junio de 2019 21:37
Para: olive-editor/olive
Cc: RafaMartorell; Author
Asunto: Re: [olive-editor/olive] Translate to Spanish and create tutorials (#1000)


Hi Rafa,

here you can see how to help translating in your language. https://github.com/olive-editor/olive/wiki/Translations

Regards, Oluja!

**Comment by RafaMartorell** [comment]
I apologize for being new and little github expert.
Here is the finished TS file, I used the code "es" instead of "es_ES" because the translation I have done is common for all the countries that speak Spanish.
I notice that the Qt Linguist app automatically put version = "2.0" in place of a larger number than the original ts, which is "2.1", although this is easy to solve.
Please do not laugh, but I just discovered that I can upload the file here :-D
Yesterday in my eagerness to offer the translation to the developers I do not know if I took the right steps, so I apologize again if it was not like that.
[olive-es_ts-file.zip](https://github.com/olive-editor/olive/files/3342714/olive-es_ts-file.zip)

**Comment by RafaMartorell** [comment]
I have completed version 2.1 and corrected some spelling mistakes, this version is finished and revised.
[olive_es.ts-files.zip](https://github.com/olive-editor/olive/files/3342946/olive_es.ts-files.zip)

**Comment by RafaMartorell** [comment]
I am a perfectionist and I observed that some words remained untranslated so I looked for lines of several ts files until I got a 100% translation in Spanish. I think that this ts file is the definitive one to have the Alpha version totally in Spanish.
The reason to change "es_ES" to "es" is because this translation is valid for any Spanish-speaking country and not only for Spain.
[olive_es.ts.zip](https://github.com/olive-editor/olive/files/3343120/olive_es.ts.zip)
A pleasure to collaborate with this project and excuse my lack of programming knowledge, I work in the audiovisual field.
